devops/scripts/aad/aad-app-reg.sh

```diff
@@ -361,16 +361,15 @@ echo "Setting API permissions (email / profile / ipaddr)"
 az ad app permission add --id "${apiAppId}" --api "${msGraphAppId}" \
   --api-permissions ${msGraphEmailScopeId}=Scope ${msGraphOpenIdScopeId}=Scope ${msGraphProfileScopeId}=Scope
 
-# todo: [Issue 1352](https://github.com/microsoft/AzureTRE/issues/1352)
-# echo "Updating redirect uri"
-# Update app registration with redirect urls (SPA)
-# az rest --method PATCH --uri "${msGraphUri}/applications/${apiAppObjectId}" \
-#     --headers 'Content-Type=application/json' \
-#     --body '{"spa":{"redirectUris":["https://localhost:8080"]}}'
-
 # Make the current user an owner of the application.
 az ad app owner add --id "${apiAppId}" --owner-object-id "$currentUserId"
 
+# Add the application admin service principal as an owner of the workspace app
+if [[ $workspace -eq 1 ]]; then
```

Comment:
Will this get called? Isn't this in a conditional block only being run for API as opposed to Workspace?

Reply:
I don't think it runs only for the API and anyway I run this multiple times and it always created the app with the right owner added.